cmake/modules/contrib/TensorRT.cmake

```diff
@@ -27,10 +27,18 @@ if(USE_TENSORRT)
     if(NOT TENSORRT_FOUND)
         message(ERROR "Could not find TensorRT.")
     endif()
-    file(GLOB TENSORRT_SRCS src/contrib/subgraph/*.cc)
     include_directories(${TENSORRT_INCLUDE_DIR})
-    list(APPEND RUNTIME_SRCS ${TENSORRT_SRCS})
     list(APPEND TVM_RUNTIME_LINKER_LIBS ${TENSORRT_LIB_DIR})
+
+    # NNVM TRT sources
```

Comment:
Do we still need NNVM?

Reply:
If a customer has a compiled model that has NNVM/TRT subgraphs, then that model would no longer work if we removed the NNVM/TRT runtime. So we need to keep it in order to be backwards compatible with all neo compiled models.

I think its best to keep both for now while we are transitioning from NNVM to Relay. We can later remove it and give an error telling users to recompile their model in order to utilize relay TRT.

Reply:
Now that upstream has removed NNVM, it'll be hard for us to keep it in neo-ai against possible future refactors.

Reply:
I think it is possible to keep NNVM/TRT runtime only even with NNVM removed, I'll look into it.